src/includes/performance-monitoring/configuration/javascript.mdx

```diff
@@ -235,15 +249,17 @@ Sentry.init({
 
 **Retrieving a Transaction**
 
-In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction otherwise it returns `undefined`. If you are using our Tracing integration by default we attach the Transaction to the Scope. So you could do something like this:
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction on the scope, otherwise it returns `undefined`. If you are using our BrowserTracing integration by default we attach the Transaction to the Scope. So you could do something like this:
```

Comment:
```suggestion
In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction on the scope, otherwise it returns `undefined`. If you are using our BrowserTracing integration, by default we attach the Transaction to the Scope. So you could do something like this:
```

Also - do we attach all transactions to the scope automatically when using the `BrowserTracing` integration, or just the `pageload` and `navigation` ones?

Reply:
We only attach `pageload` and `navigation` transaction to the scope. I can mention that.